firebase-docs.md

```diff
@@ -0,0 +1,217 @@
+  - [Additional Configuration](#additional-configuration)
+      - [Firebase Storage](#firebase-storage)
+  - [Set up your secrets:](#set-up-your-secrets)
+    - [Import JSON Data Using the GUI](#import-json-data-using-the-gui)
+  - [File Structure](#file-structure)
+
+# Installation
+## Requirements
+
+Before you start setting up Firebase and pushing dummy data, ensure you meet the following requirements:
+
+### Firebase Services
+
+1. **Google Account**: You need a Google account to create and manage Firebase projects.
+2. **Get your Firebase on.** This means creating a Firebase account and downloading the JSON key file.
+
+- Go to the Firebase console: https://console.firebase.google.com/
+- Click on the **Add project** button.
+- Enter a name for your project and click on the **Continue** button.
+- Select the Google Cloud Platform project to associate with your Firebase project, or create a new one.
+- Select the location where you want to store your Firebase project data.
+- Click on the **Create project** button.
+- Once your project is created, click on the **Project settings** tab.
+- Click on the **Service accounts** tab.
+- Click on the **Generate new private key** button.
+- Click on the **Generate Key** button.
+- A file named `service-account-file.json` will be downloaded to your computer.
+- Save the downloaded JSON file in the `firebase/res/` directory.
+
+### API Keys
+
+#### OpenAI API Key
+
+- Go to the OpenAI platform: [platform.openai.com](https://platform.openai.com/)
+- Click on the Sign in button.
+- Sign in with your OpenAI account.
+- Click on the API Keys tab.
+- Click on the Create new API key button.
+- Copy the API key and save it in a safe place.
+
+#### Web3.Storage API Key
+
+- Go to Web3 Storage: [web3.storage](https://web3.storage)
+- Click on the Sign up button.
+- Create a Web3 Storage account.
+- Click on the API keys tab.
+- Click on the Create new API key button.
+- Copy the API key and save it in a safe place.
+
+## Additional Configuration
+
+Before initializing Firebase and pushing dummy data, perform the following additional configurations:
+
+#### Firebase Storage
+
+**Enable Firebase Storage**: In the Firebase Console, navigate to your project and go to "Storage." Follow the prompts to enable Firebase Storage.
+- Go to the Firebase console: https://console.firebase.google.com/
+- Click on your project.
+- Click on the **Storage** tab.
+- Click on the **Get started** button.
+- Follow the instructions to enable Firebase Storage.
+
+## Set up your secrets:
+- This involves creating a new Python file called `secrets.py` and adding the following code:
+
+   ```python
+
+   import os
+
+   dir_path = os.path.dirname(os.path.realpath(__file__))
+   SERVICEACCOUNTFILE = os.path.join(dir_path, "firebase", "res", "service-account-file.json")
+   api_key = "sk-asjkdsakdasasdklldas"  # OPENAI KEY
+   dbURL = "https://codemark-jdxknzklndkasnkldjkasljd.firebasedatabase.app/"
+   firebaseConfig = {
```

Comment:
for firebaseConfig, user need to go to Project Setting => General => Scroll to bottom Section => Your Apps => Add new App in web => copy the key, values and place them selectively in firebaseConfig


please databaseURL value in dbURL and then reference it here!